src/tests/ftest/util/server_utils.py

```diff
@@ -205,15 +205,6 @@ def prepare(self, storage=True):
         # Clean up any files that exist on the hosts
         self.clean_files()
 
-        if storage:
-            # Prepare server storage
-            if self.manager.job.using_nvme or self.manager.job.using_dcpm:
-                self.log.info("Preparing storage in <format> mode")
-                self.prepare_storage("root")
-                if hasattr(self.manager, "mca"):
-                    self.manager.mca.update(
-                        {"plm_rsh_args": "-l root"}, "orterun.mca", True)
```

Comment:
If the `network/cart_self_test.py` test fails - one of the few tests that still use `orterun` to launch the server - then we may need to reinstate this code.  If that does end up being the case, I'd suggest something like this: 
```
        if self.manager.job.using_nvme or self.manager.job.using_dcpm:
            if hasattr(self.manager, "mca"):
                self.manager.mca.update({"plm_rsh_args": "-l root"}, "orterun.mca", True)
```